commands/configure_director_test.go

```diff
@@ -117,6 +118,132 @@ var _ = Describe("ConfigureDirector", func() {
 			Expect(logger.PrintfArgsForCall(11)).To(Equal("finished configuring resource options for bosh tile"))
 		})
 
+		Context("when the --config flag is set", func() {
+			Context("when other flags are set", func() {
+				It("returns an error", func() {
+					err := command.Execute([]string{"--config", "test.yml", "--az-configuration", "{}"})
+					Expect(err).To(MatchError("config flag can not be passed with another configuration flags"))
+					err = command.Execute([]string{"--config", "test.yml", "--networks-configuration", "{}"})
+					Expect(err).To(MatchError("config flag can not be passed with another configuration flags"))
+					err = command.Execute([]string{"--config", "test.yml", "--network-assignment", "{}"})
+					Expect(err).To(MatchError("config flag can not be passed with another configuration flags"))
+					err = command.Execute([]string{"--config", "test.yml", "--director-configuration", "{}"})
+					Expect(err).To(MatchError("config flag can not be passed with another configuration flags"))
+					err = command.Execute([]string{"--config", "test.yml", "--iaas-configuration", "{}"})
+					Expect(err).To(MatchError("config flag can not be passed with another configuration flags"))
+					err = command.Execute([]string{"--config", "test.yml", "--security-configuration", "{}"})
+					Expect(err).To(MatchError("config flag can not be passed with another configuration flags"))
+					err = command.Execute([]string{"--config", "test.yml", "--syslog-configuration", "{}"})
+					Expect(err).To(MatchError("config flag can not be passed with another configuration flags"))
+					err = command.Execute([]string{"--config", "test.yml", "--resource-configuration", "{}"})
+					Expect(err).To(MatchError("config flag can not be passed with another configuration flags"))
+				})
+			})
+
+			Context("with an invalid config", func() {
+				It("configures the director", func() {
+					configYAML := `invalidYAML`
+
+					configFile, err := ioutil.TempFile("", "config.yaml")
+					Expect(err).ToNot(HaveOccurred())
+					_, err = configFile.WriteString(configYAML)
+					Expect(err).ToNot(HaveOccurred())
+					Expect(configFile.Close()).ToNot(HaveOccurred())
+
+					err = command.Execute([]string{
+						"--config", configFile.Name(),
+					})
+					Expect(err).To(HaveOccurred())
```

Comment:
@jtarchie minor nitpick, could we assert that a more specific error was received here? E.g. `MatchError` or `Expect(err.Error()).To(ContainSubstring(...))`. Just making sure we're hitting the error branch we expected to.